README.md

```diff
@@ -346,9 +346,10 @@ jobs:
 We need a use `pull_request_target` event because we need `GITHUB_TOKEN` with write permission
 to update labels, milestones of PR from forked repositories.
 
-After merged:
+After merged (unless already set):
```

Comment:
This now sounds like "default label - `maintenance`" will always be set if it has not be set before. However the three actions are checked (and acted) individually. So maybe something like `After the PR got merged each of the following actions are checked and applied if not already set:` ?